src/NodaTime.Test/Calendars/BclCalendars.cs

```diff
@@ -74,6 +74,7 @@ internal static CalendarSystem CalendarSystemForCalendar(Calendar bcl)
                 case "HebrewCalendar": return CalendarSystem.HebrewCivil;
                 case "PersianCalendar": return bcl.IsLeapYear(1) ? CalendarSystem.PersianSimple : CalendarSystem.PersianAstronomical;
                 case "UmAlQuraCalendar": return CalendarSystem.UmAlQura;
+                case "WondrousCalendar": return CalendarSystem.Wondrous;
```

Comment:
Are all the other calendars in the BCL?  I was just trying to replicate what I was seeing being done for all the NodaTime calendars.

Reply:
Yes, they are: see https://docs.microsoft.com/en-gb/dotnet/api/system.globalization?view=netframework-4.7

The purpose of this is so that we can check that our behaviour matches the BCL. Unless there's a WondrousCalendar in the BCL, we won't need it here.

Reply:
Sounds good...
So, who do I talk to get the Wondrous calendar added to the BCL?